remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -97,6 +100,13 @@ public void testGetTags() {
         assertEquals(expectedTags, actualCreatedTags);
     }
 
+    @Test
+    public void testTagNotFoundValidation()
+    {
+        given(nodeServiceMock.hasAspect(any(),any())).willReturn(true);
```

Comment:
I'm sure we don't really want to test a node that has every aspect. I think there's actually a single node and a single aspect we're interested in here. In cases like this then please can we focus on what's actually relevant to the test case?

Reply:
Remove the test case, it is no longer required.

Reply:
Why it is no longer required? Do we have any other test cases to cover those changes?

Reply:
Yes, there is a preexisting test case which cover above scenario. The test case name is "testDeleteTagById_nonExistentTag()".

Reply:
This can't really be the case though. You've made a change to the production code so either that test doesn't check your change, or perhaps it's disabled.

Reply:
Yes, we will add a new test case for the PR changes.

Reply:
Added a new test case for getTagByIdNotFound validation. Could you please review and approve the PR if no more changes required. Thanks !